### ISSUE TITLE
Ability to add row loop index as column

### ISSUE THREAD

**Issue by djsegal** [offirgolan/ember-light-table#70]
I'd like to have a first column that just adds an index to each row signifying the order models were taken from the database

equivalent to `index` in the following code snippet:

```
{{#each users as |user index|}}
  <tr>
    <td>{{index}}</td>
    <td>{{user.name}}</td>
    ...
  </tr>
{{/each}}
```

-------

I couldn't pick up from the docs wether this was currently possible or not

**Comment by offirgolan** [comment]
You can just add the index as a property on the model and make a column for it.

```js
users.forEach((user, index) => user.set('_index', index));
```

Then add the following column

```js
{ 
  label: 'Index',
  valuePath: '_index'
}
```

**Comment by djsegal** [comment]
pretty dope trick. thanks man

**Comment by offirgolan** [comment]
No problem! Glad I could help 😸

On Thursday, June 23, 2016, djsegal <notifications@github.com> wrote:

> Closed #70 <https://github.com/offirgolan/ember-light-table/issues/70>.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/offirgolan/ember-light-table/issues/70#event-702175966>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAjJwh0cdVdcLzac1P9ueOwBsATygA6cks5qOrtugaJpZM4I8W-2>
> .
>


-- 
Thanks,
Offir Golan
